### ISSUE TITLE
Upgrading to 0.3.15 -> 0.3.22 

### ISSUE THREAD

**Issue by bruno-trindade** [pboling/flag_shih_tzu#77]
After upgrading, it seems like there may have been a major change in the flag retrieval because models that were returning "true" for this particular flag were now all uniformly returning "false" for that flag.  

Here is the flag setup on the model:

```ruby
has_flags 1 => :cje,
    2 => :apq,
    3 => :pje,
    4 => :ums,
    5 => :esp,
    6 => :avl,
    7 => :mpop,
    8 => :cvb,
    9 => :cob,
    10 => :cje,
    11 => :sue,
    12 => :lpl,
    13 => :ppl,
    14 => :lsms,
    15 => :lfwp,
    16 => :lfwop,
    17 => :ftp,
    18 => :plu,
    19 => :mpl,
    20 => :mat,
    21 => :skq,
    22 => :tma,
    23 => :rnw,
    flag_query_mode: :bit_operator,
    check_for_column: false
```

The flag in question is the **last** one, `:rnw` - all the others seem to be in the correct state.

**Comment by bruno-trindade** [comment]
Closing - I've determined the problem was unrelated to the gem!

**Comment by pboling** [comment]
That is a lot of flags!! 🥇 The most I have ever seen.

**Comment by pboling** [comment]
@bruno-trindade Just released 0.3.23 with a minor bugfix: #76

**Comment by bruno-trindade** [comment]
Thank you @pboling - appreciate the note on the release!
